EmployeesSalaries/EmployeesSalaries/Models/SalaryCalculaters/DevSalary.cs

```diff
@@ -0,0 +1,12 @@
+﻿using EmployeesSalaries.Models.Interfaces;
+
+namespace EmployeesSalaries.Models
+{
+    public class DevSalary : IEmployeeSalaryCalculater
+    {
+        public double BasicSalary { get; set; } = 800;
+
+        public double GetTotalSalary() { return BasicSalary; }
+        public bool IsRequiredCalculater(string Role) { return Role == "Dev"; }
```

Comment:
don't use strings to check for types or roles

Reply:
Removed Strings and role